libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/PredictiveCacheController.kt

```diff
@@ -55,19 +55,22 @@ private const val MAX_ZOOM = 16.toByte()
  * - `OnboardRouterOptions` enabled you to specify a path where nav-tiles will be saved and if a
  * custom directory was used, it should be cleared as well.
  *
- * @param predictiveCacheLocationOptions [PredictiveCacheLocationOptions] location configuration for navigation predictive caching
+ * @param predictiveCacheLocationOptions [PredictiveCacheLocationOptions] location configuration for visual map predictive caching (optional)
+ * @param predictiveCacheGuidanceLocationOptions [PredictiveCacheLocationOptions] location configuration for guidance predictive caching (optional)
  * @param predictiveCacheControllerErrorHandler [PredictiveCacheControllerErrorHandler] listener (optional)
  */
 class PredictiveCacheController @JvmOverloads constructor(
     private val predictiveCacheLocationOptions: PredictiveCacheLocationOptions =
         PredictiveCacheLocationOptions.Builder().build(),
+    private val predictiveCacheGuidanceLocationOptions: PredictiveCacheLocationOptions =
+        PredictiveCacheLocationOptions.Builder().build(),
```

Comment:
```suggestion
    private val predictiveCacheGuidanceLocationOptions: PredictiveCacheLocationOptions =
        predictiveCacheLocationOptions,
```
This should default to the previous options to retain backwards compatibility. Let's also document that clearly.